src/TransloaditClient.js

```diff
@@ -394,7 +522,7 @@ class TransloaditClient {
     form.append('params', jsonParams)
 
     if (fields == null) {
-      fields = []
+      fields = {}
```

Comment:
fields is expected as an object, so making it an array by default is somewhat misleading/confusing

Reply:
👍 👍